things/postgres/things_test.go

```diff
@@ -358,10 +365,24 @@ func TestMultiThingRetrieval(t *testing.T) {
 			limit:  n,
 			size:   0,
 		},
+		"retrieve things with valid name": {
+			owner:  email,
+			offset: 0,
+			limit:  n,
+			name:   name,
+			size:   1,
+		},
+		"retrieve things with invalid name": {
```

Comment:
This is not an invalid name. There are no things with this name. Think about adding an invalid name use case (with a name which length is larger than the limit that you've set).